shopfloor_mobile_base/static/wms/src/router.js

```diff
@@ -11,30 +11,34 @@ import {Language} from "./settings/language.js";
 import {Workstation} from "./settings/workstation.js";
 import {LoginPage} from "./loginpage.js";
 import {process_registry} from "./services/process_registry.js";
+import {page_registry} from "./services/page_registry.js";
 // Const NotFound = { template: '<div>Lost in the scanner app.</div>' }
 
 // TODO: handle routes via registry
 let routes = [
     {path: "/", component: HomePage, name: "home"},
     {path: "/login", component: LoginPage, name: "login"},
     {path: "/settings", component: SettingsControlPanel, name: "settings"},
-    {path: "/profile", component: Profile, name: "profile"},
```

Comment:
Why profile is not part of this module ?

Reply:
Because is not needed. See `page_registry` usage. The only required page here is `/settings` which will collect settings page automatically.